src/domains/community/detail/page/DetailCommunityPage.jsx

```
@@ -31,13 +32,16 @@ const DetailCommunityPage = () => {
   const currentUrl = window.location.href
   const hasSaved = useRef(false)
   const hasFetched = useRef(false)
+
   const {
+    title,
+    setTitle,
     content,
     setContent,
     loading: editLoading,
     error: editError,
```

ESLint

[error] 'editError' is assigned a value but never used.

(no-unused-vars)